### ISSUE TITLE
Redirection incorrectly trying to reach REST at HTTPS URL

### ISSUE THREAD

**Issue by ih8gates** [johngodley/redirection#705]
I discovered today that Redirection has stopped working. I get the "Something went wrong" screen.

I can see that  Redirection is having trouble reaching the REST API URL. 
The weird thing is that it's trying to reach the HTTPS version of the URL when the site settings don't have HTTPS in the URL. 

Is there a way I can make Redirection look at the normal HTTP variation of the URL?

```
Plugin: 3.1
WordPress: 4.9.2 (single)
PHP: 7.0.27
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36
REST API: https://essmarketing.webfactional.com/wp-json/

Error: Failed to fetch

Error: Failed to fetch
```

**Comment by ih8gates** [comment]
I was able to get Redirection working again by reverting to an older version.

**Comment by johngodley** [comment]
This should now be fixed in 3.1.1

**Comment by ih8gates** [comment]
Confirmed. 3.1.1 is working for me.
